src/kbmod/filters/stamp_filters.py

```diff
@@ -103,7 +103,7 @@ def keep_row(self, row: ResultRow):
             return False
 
         # Find the peak in the image.
-        stamp = row.stamp.reshape([self.width, self.width])
+        stamp = row.stamp
```

Comment:
Is this change because stamp is already in the correct shape? Or is there some other reason?

Reply:
yep!